src/utils/convert-filter.js

```diff
@@ -10,7 +9,7 @@ const convertFilter = (filter) => {
     switch (property.type()) {
     case 'string':
       return {
-        [property.name()]: { [Op.like]: `%${escape(value)}%` },
+        [property.name()]: { [Op.like]: `%${value}%` },
```

Comment:
we cannot do it like this because when someone gives "%" in the input it won't be escaped.